x/interchaintxs/types/errors.go

```diff
@@ -9,5 +9,8 @@ var (
 	ErrInvalidICAOwner           = sdkerrors.Register(ModuleName, 1100, "invalid interchain account interchainAccountID")
 	ErrInvalidAccountAddress     = sdkerrors.Register(ModuleName, 1101, "invalid account address")
 	ErrInterchainAccountNotFound = sdkerrors.Register(ModuleName, 1102, "interchain account not found")
-	ErrNotContract               = sdkerrors.Register(ModuleName, 1113, "not a contract")
+	ErrNotContract               = sdkerrors.Register(ModuleName, 1103, "not a contract")
```

Comment:
![image](https://user-images.githubusercontent.com/24505352/187265848-8963fb67-0467-4b74-8297-bb6a948be7ca.png)
you sure that's ok?

Reply:
Yes, different modules can have same code errors, because uniqueness is _(module, code)_ combo